### DISCUSSION TITLE
Importing RSS Feeds from File

### DISCUSSION THREAD

**Discussion by tokisuno** [koreader/koreader#10518] [Ideas]
I use newsboat on my desktop, and I was wondering if there would be any way to be able to feed KOReader a plain-text file that only contains urls and make a feed off of that? 

Obviously this could be expanded upon by separating the urls from each other with commas, or even formatting a simple json file.

**Comment by Frenzie** [comment]
Not at present, but for anyone interested in implementing something like that, note that the standard method would be OPML.
